### PR TITLE
Allow fetch prerelease

### LINKED ISSUE #494: Get Latest Pre-Release
It seems that there is no way to get the latest pre-release from a repository.

#269 seems to hint at this, but rather asks for only *non* pre-releases, as opposed to *only* pre-releases

### PR DESCRIPTION
**Pull request by crossan007**
closes #494

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you for this PR. I like that you added both documentation and test. 
Correct me if Im wrong but this allows the user to do: 

```php
$latestRelease = $api->latestIncludingPrereleases('username', 'repo');
```

Wouldn't the result be the same if you did: 
```php
$latestRelease = $api->all('username', 'repo')[0];
```

If so, I do not believe we need this PR. The API client should reflect the API and the API does not have a endpoint for getting the latest release including prereleases and drafts. 

Other maintainers/contributors may have different opinion though?

**Comment by crossan007** [comment]
Thanks for your reply - Your interpretation is correct.

I just thought it would be nice to have an aptly named function to access the requested data.

If nothing else, this PR (and issue #494) will serve as a reference for other users of this project who may want to access pre-releases within their application.

**Comment by Nyholm** [comment]
What do you think about just updating the documentation and not the code? The documentation can show how you get the latest release including prereleases.

**Comment by crossan007** [comment]
Sure.   I can update this PR

File: doc/repo/releases.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ Provides information about releases for a repository. Wraps [GitHub Releases API
 $release = $client->api('repo')->releases()->latest('twbs', 'bootstrap');
 ```
 
+The ```latest()``` method fetches only releases which are not marked "prerelease."
```
Comment (Nyholm) [inline doc/repo/releases.md]: You should only use one backtick at each side of the word. 
\`latest()\`

Comment (Nyholm) [inline doc/repo/releases.md]: "prerelease" and "draft".

File: doc/repo/releases.md [hunk 2]
```diff
@@ -9,6 +9,12 @@ Provides information about releases for a repository. Wraps [GitHub Releases API
 $release = $client->api('repo')->releases()->latest('twbs', 'bootstrap');
 ```
 
+The ```latest()``` method fetches only releases which are not marked "prerelease."  
+
+To obtain the latest release *including prereleases*, select the first element in the "all releases" function:
```
Comment (Nyholm) [inline doc/repo/releases.md]: \*including\* prereleases and drafts,

File: doc/repo/releases.md [hunk 3]
```diff
@@ -9,6 +9,12 @@ Provides information about releases for a repository. Wraps [GitHub Releases API
...
+```
```
Comment (Nyholm) [inline doc/repo/releases.md]: \`\`\`php

File: doc/repo/releases.md (hunk 2 above)
Comment (crossan007) [inline doc/repo/releases.md]: auth is required for drafts.  I've added that to the doc also

File: doc/repo/releases.md [hunk 4]
```diff
@@ -9,6 +9,15 @@ Provides information about releases for a repository. Wraps [GitHub Releases API
 $release = $client->api('repo')->releases()->latest('twbs', 'bootstrap');
 ```
 
+The `latest()` method fetches only releases which are not marked "prerelease" or "draft"
```
Comment (Nyholm) [inline doc/repo/releases.md]: Missing period.

Comment (Nyholm) [inline doc/repo/releases.md]: Bump, can you fix this minor so I can merge?

**Comment by Nyholm** [comment]
Closed by #519

**Pull request closed without merging**